5.1.vinaigrette.py

```diff
@@ -0,0 +1,17 @@
+import random
+from datetime import datetime, timedelta
+
+date_in1 = input("Enter the first date in the format YYYY-MM-DD: ")
```

Comment:
You should organize your code in functions.
It's bad practice leaving code parts unorganized